Cleaner integration of Prettier & ESLint

indeyets opened the pull request:
This PR makes sure, that JS-related Prettier rules are run as part of regular ESLint checks. It means that devs don't
have to run 2 separate commands.

davidmz commented on .eslintrc.js:
```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: 'eslint:recommended',
+  extends: ['eslint:recommended', 'prettier', 'prettier/react'],
```
Я правильно понимаю, что eslint сам теперь будет вызывать prettier и, например, в IDE не надо отдельно включать последний?

indeyets replied:
Да!

indeyets committed:
Integrate prettier into eslint

indeyets committed:
Get rid of potentially-conflicting rules

indeyets committed:
Disable husky/pretty-quick

We run checks on Travis and users should be free to configure local
checks the way they like (IDE, manual checks before push, etc.)

indeyets committed:
Upgrade ESLint & Prettier
